File-Formats/PDF-to-image/Converting-PDF-pages-into-Images.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Converting PDF pages into images| Syncfusion
+description: Learn about Converting PDF pages into images using Syncfusion PdfToImageConverter.
+platform: file-formats
+control: PDF to image
+documentation: UG
+---
+
+# Converting PDF pages into images
+
+There are various types of convert operations in Syncfusion PdfToImageConverter. Please specific operations are documented under below re-directions.
```

Comment:
PdfToImageConverter allows you to convert pages from a PDF document into images. The convertion opertations for different platforms are documented under below redirection links.

Reply:
Yes. I have done the changes.